src/visitors/components/ActivitiesGridItem.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { PrimaryButtonNoFill, SecondaryButton } from '../../shared/components/form';
+
+
+const BtnOne = styled(PrimaryButtonNoFill)`
+  width: 100%;
+  height: 100%;
+  padding: 2em 0;
+`;
+
+const BtnTwo = styled(SecondaryButton)`
+  width: 100%;
+  height: 100%;
+  padding: 2em 0;
+`;
+
+const ActivitiesGridItem = props => (
+  <>
+    {
+      props.index % 2 === 0
+        ? (
+          <BtnOne name={props.name} onClick={e => props.onClick(e.target.name)}>
```

Comment:
This layout is a bit difficult to read. Is it possible for the component to live on the same line as the ternary or is this an eslint thing?

Reply:
It's a linter thing, max line limit